packages/o/opensnitch/pspec_x86_64.xml

```diff
@@ -0,0 +1,201 @@
+<PISI>
+    <Source>
+        <Name>opensnitch</Name>
+        <Homepage>https://github.com/evilsocket/opensnitch</Homepage>
+        <Packager>
+            <Name>Robert Gonzalez</Name>
+            <Email>uni.dos12@outlook.com</Email>
+        </Packager>
+        <License>GPL-3.0-or-later</License>
+        <PartOf>network.util</PartOf>
+        <Summary xml:lang="en">OpenSnitch is a GNU/Linux application firewall.</Summary>
+        <Description xml:lang="en">OpenSnitch is a GNU/Linux interactive application firewall inspired by Little Snitch. 
+</Description>
+        <Archive type="binary" sha1sum="79eb0752a961b8e0d15c77d298c97498fbc89c5a">https://sources.getsol.us/README.Solus</Archive>
+    </Source>
+    <Package>
+        <Name>opensnitch</Name>
+        <Summary xml:lang="en">OpenSnitch is a GNU/Linux application firewall.</Summary>
+        <Description xml:lang="en">OpenSnitch is a GNU/Linux interactive application firewall inspired by Little Snitch. 
+</Description>
+        <PartOf>network.util</PartOf>
+        <Files>
+            <Path fileType="config">/etc/opensnitchd/default-config.json</Path>
+            <Path fileType="config">/etc/opensnitchd/rules</Path>
+            <Path fileType="config">/etc/opensnitchd/system-fw.json</Path>
```

Comment:
We're going to want these to not be installed to `/etc` in order to be stateless. If the files are meant to be edited or overwritten, they should install to `/usr/share/defaults/etc/opensnitchd`. Some code may need to be added to the application to look there for files as a fallback for anything in `/etc`.